tests/test_wave_01.py

```diff
@@ -74,12 +74,12 @@ def test_adds_movie_to_user_watched():
     updated_data = add_to_watched(user_data, movie)
 
     # Assert
-    assert len(updated_data["watched"]) is 1
+    assert len(updated_data["watched"]) == 1
     assert updated_data["watched"][0]["title"] is MOVIE_TITLE_1
```

Comment:
Great that you corrected to `==` on line 77; We'd want to do the same for the following three lines!